.gitignore

```diff
@@ -8,3 +8,20 @@ venv/
 **__pycache__/
 *.pyc
 **db.sqlite3
+*.jpg
+*.jpeg
+*.jpe
+*.jif
+*.jfif
+*.jfi
+*.jp2
+*.j2k
+*.jpf
+*.jpx
+*.jpm
+*.mj2
+*.png
+*.jxr
+*.hdp
+*.wdp
+*.webp
```

Comment:
can you add just a dir, where media will be stored ?

Reply:
same mistake as in my prev task)